ansible/roles/ci-base-setup/tasks/freebsd.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Use of raw removed.  This was so it could be run without python being installed, but that's (1) insufficient and (2) python is installed but with a specific version, which the ci-jenkins-freebsd group has been set to use by default.